modules/system/config-wfrp4e.js

```diff
@@ -1174,13 +1174,14 @@ WFRP4E.PrepareSystemItems = function() {
                             let skillName = game.i18n.localize("NAME.Cool");
                             let test = await args.actor.setupSkill(skillName, {terror: true, appendTitle : " - Terror"});
                             await test.roll();
-                            await this.actor.applyFear(terror, name)
                             if (test.failed)
                             {
                                 if (test.result.SL < 0)
                                     terror += Math.abs(test.result.SL)
 
                                 await this.actor.addCondition("broken", terror)
+                            } else {
+                                await this.actor.applyFear(terror, name)
```

Comment:
This is incorrect, fear should always be applied.

Reply:
that's interesting, in Polish book, it says that the fear should only be applied after passing the Terror test.